parametric_stellarator.py

```diff
@@ -353,9 +379,10 @@ def stellarator_torus(
 
 def parametric_stellarator(
     plas_eq, num_periods, radial_build, gen_periods, num_phi = 60,
-    num_theta = 100, exclude = [], step_export = True, h5m_export = None,
-    plas_h5m_tag = None, include_graveyard = False, facet_tol = None,
-    len_tol = None, norm_tol = None, min_mesh_size = 5.0, max_mesh_size = 20.0,
+    num_theta = 100, exclude = [], plas_h5m_tag = None,
```

Comment:
We should discuss strategies to manage this ballooning list of arguments, possibly with 1 or more dictionaries